### PR TITLE
feat(*): Add `arrayName functionality

### PR DESCRIPTION
**Pull request by BBlackwo**
As discussed in  #11, added functionality to specify array name of subscriptions.

Also did a bit of clean up. Let me know if I need to put some of these things back.

### CODE REVIEW

**Comment by BBlackwo** [comment]
Wasn't sure if you wanted me to update the version or if you would do that.

**Comment by NetanelBasal** [comment]
Thanks, looks really good. We need to get back the yarn.lock file and you can also bump the version. ( 2.4.0 )

File: src/auto-unsubscribe.ts [hunk 1]
```diff
@@ -1,8 +1,14 @@
-function isFunction( fn ) {
-  return typeof fn === 'function';
+const isFunction = (fn) => typeof fn === 'function';
```
[the remark this answers is not in the document]
Comment (BBlackwo) [reply src/auto-unsubscribe.ts]: Good spot. Fixed.

**Comment by BBlackwo** [comment]
@NetanelBasal I bumped the version. It looked like the `yarn.lock` file wasn't being used as there's also a `package-lock.json`. Can we used one or the other?

**Comment by NetanelBasal** [comment]
Closes #15.

**Pull request merged**